docs/user-guide.md

```diff
@@ -200,6 +200,7 @@ The currently supported options are:
 | kompose.service.healthcheck.liveness.http_get_path  | kubernetes liveness httpGet path                                                     |
 | kompose.service.healthcheck.liveness.http_get_port  | kubernetes liveness httpGet port                                                     |
 | kompose.service.healthcheck.liveness.tcp_port       | kubernetes liveness tcpSocket port                                                   |
+| kompose.service.external-traffic-policy: local       | kubernetes service external traffic policy                                                   |
```

Comment:
```suggestion
| kompose.service.external-traffic-policy       | 'cluster', 'local', ''                                                   |
```

Let's just do this for now until we update the documentation on user guide. It's a bit of a mess right now to understand / comprehend and I plan to work on this, this month.